client/blocks/nps-survey/docs/example.jsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import NpsSurvey from '../';
+import { isEnabled } from 'config';
+
+class NpsSurveyExample extends PureComponent {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			wasDismissed: false,
```

Comment:
A properly written constructor leaves the resulting object in a valid state. In this case it'd be valid without explicit initialization but I also like the overview of state props.